README.md

```diff
@@ -18,313 +18,10 @@ Even though cargo-dist is primarily a tool for building and packaging applicatio
 
 That said, the current version is Very Very Unstable And Experimental and the extra conveniences currently only work with Github CI and Github Releases™️!
 
-* [Way-Too-Quick Start](#way-too-quick-start)
-* [Installation](#installation)
-  * [Install Prebuilt Binaries With cargo-binstall](#install-prebuilt-binaries-with-cargo-binstall)
-  * [Build From Source With Cargo](#build-from-source-with-cargo)
-  * [Download Prebuilt Binaries From Github Releases](#download-prebuilt-binaries-from-github-releases)
-  * [Use The Installer Scripts](#use-the-installer-scripts)
-* [Setup](#setup)
-  * [Configuring Installers](#configuring-installers)
-  * [Configuring Targets](#configuring-targets)
-* [Usage (CI)](#usage-ci)
-* [Usage (Local Builds)](#usage-local-builds)
-* [Concepts](#concepts)
-* [Build Flags](#build-flags)
-* [Compatibility With Other Tools](#compatibility-with-other-tools)
-* [Contributing](#contributing)
-  * [Updating Snapshots](#updating-snapshots)
-  * [Cutting Releases](#cutting-releases)
+# Read The Book!
 
-# Way-Too-Quick Start
+We've got all the docs you need over at the [cargo-dist book](https://axodotdev.github.io/cargo-dist/book/)!
```

Comment:
It might be add some nav hints,  say like "if you are starting out and want the simple start go here; if you know you have complicated set up and want to dive into custom config right away go here"